recipes/napari-n2v/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "napari-n2v" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-n2v-{{ version }}.tar.gz
+  sha256: 476c39dd4ed66c99c2a0a76cba6a472090c253d4e4a9aa6648a0af409ed16400
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bioimageio.core
+    - imageio !=2.11.0,!=2.22.1,>=2.5.0
+    - magicgui
+    - n2v >=0.3.2
+    - napari <=0.4.15
+    - napari-time-slicer >=0.4.9
+    - numpy
+    - pyqtgraph
+    - python >=3.7
+    - qtpy
+    - scikit-image
```

Comment:
Shouldn't this also have an explicit dependency on tensorflow? It will still fail on Windows in that case, but at environment build time rather than on import.

Reply:
You are right!